src/minter-contract-utils/src/operation_store.rs

```diff
@@ -270,6 +273,27 @@ where
             log::trace!("Operation {id} is evicted from the operation log");
         }
     }
+
+    /// Retrieves all operations for the given ETH wallet address, starting from `offset` returning a max of `count` items
+    /// If `offset` is `None`, it starts from the beginning.
+    /// If `count` is `None`, it returns all operations.
+    fn operations_for_address(
```

Comment:
This fn covers all use cases. Remove `get_page_for_address` and `get_for_address`